compiler-plugin/src/main/java/io/ballerina/transformer/plugin/TransformerCodeValidator.java

```diff
@@ -206,6 +210,13 @@ private boolean isTransformerFunc(FunctionDefinitionNode funcDefNode) {
                 && funcDefNode.functionBody().kind() == SyntaxKind.EXPRESSION_FUNCTION_BODY;
     }
 
+    private boolean isTransformerFuncAllowConcurrentCalls(FunctionDefinitionNode funcDefNode) {
+        return !funcDefNode.qualifierList().isEmpty() &&
+                funcDefNode.qualifierList().stream().anyMatch(qualifier ->
+                        qualifier.kind() == SyntaxKind.ISOLATED_KEYWORD)
+                && funcDefNode.functionBody().kind() == SyntaxKind.EXPRESSION_FUNCTION_BODY;
+    }
```

Comment:
Can't we simplify the logic as below?
```suggestion
    private boolean isIsolated(FunctionDefinitionNode funcDefNode) {
        return funcDefNode.qualifierList().stream().anyMatch(qualifier ->
                        qualifier.kind() == SyntaxKind.ISOLATED_KEYWORD);
    }
```

Reply:
Yeah, it can be